stdlib/public/core/StringBridge.swift

```diff
@@ -177,6 +179,7 @@ extension String {
       count: length,
       elementShift: isUTF16 ? 1 : 0,
       hasCocoaBuffer: true,
+      hasNulTerminator: false,
```

Comment:
@phausler of course null termination is expected, but we don't yet support `CFStringGetCStringPtr`, so it's not a problem.